clash/dragoncurse_castle.tex

```diff
@@ -33,16 +33,15 @@ \subsection*{\MakeUppercase{Player Setup}}
 \textbf{Town Buildings:} \svgunit{bronze} Dwelling
 
 \textbf{Map Tile Pool:} None
-
-\textbf{Additional Bonus:} None
+\vspace*{\fill}\columnbreak
```

Comment:
This is fine, but there's an uncomfortably big blank space now. May I suggest putting some art here? I think land mine would work fine. Please use the transparency package with this, setting the value to 0.3 (see how it's done in the bloody grail art).

![land_mine](https://github.com/user-attachments/assets/0b1236d3-439d-4bd6-9e58-62fc7a3fe40d)